packages/storybook/src/spright/chat/conversation/chat-conversation.stories.ts

```diff
@@ -44,7 +50,34 @@ export const chatConversation: StoryObj<ChatConversationArgs> = {
                 ></${spinnerTag}>
             </${chatMessageTag}>
             <${chatMessageTag} message-type="${() => ChatMessageType.inbound}">
+                <${buttonTag} slot='footer-actions' appearance='ghost' content-hidden>
```

Comment:
The [nimble-button accessibility guidance](https://nimble.ni.dev/storybook/?path=/docs/components-button--docs#accessibility) suggests also setting a `title` whenever you set content-hidden. I think that makes sense for this use case. Could you update the user-visible stories to do this for each of these buttons? (Just the ones that are content-hidden, not the ones with visible text content)

I considered also asking for you to document this suggestion in the slot's docs, but I think I'd rather not duplicate the guidance that we already have in the button docs.